PacakingProjects/BuildAgentPayload/BuildAgentPayload.csproj

```diff
@@ -0,0 +1,76 @@
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{ECEADD3A-5992-420F-950D-510BBDDC06E1}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup>
+    <SourcesRoot>$(MSBuildThisFileFullPath)\..\..\..</SourcesRoot>
+    <DestinationDir>$(SourcesRoot)\DeploymentArtifacts\BuildAgentPayload\$(Configuration)\</DestinationDir>
+    <!-- OutputPath is required by the Clean and Rebuild tasks --><!--
+    <OutputPath>$(DestinationDir)</OutputPath>-->
+  </PropertyGroup>
+  <ItemGroup>
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\SonarQube.Common.dll" />
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\SonarQube.MSBuild.Runner.exe" />
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\SonarQube.MSBuild.Runner.exe.config" />
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.MSBuild.Tasks\bin\$(Configuration)\Targets\SonarQube.Integration.ImportBefore.targets" />
+  </ItemGroup>
+  <ItemGroup Condition="$(Configuration)== 'Debug'">
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\SonarQube.Common.pdb" />
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\SonarQube.MSBuild.Runner.pdb" />
+  </ItemGroup>
+  <!-- Reference to dependencies to ensure the build order is correct -->
+  <ItemGroup>
+    <ProjectReference Include="..\..\SonarQube.Bootstrapper\SonarQube.Bootstrapper.csproj">
+      <Project>{60aacbb4-1661-4eeb-a029-91289c1d3f7e}</Project>
+      <Name>SonarQube.Bootstrapper</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\SonarQube.Common\SonarQube.Common.csproj">
+      <Project>{e312fdbb-bc13-4559-8f21-76ca5b88ebc5}</Project>
+      <Name>SonarQube.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\SonarQube.MSBuild.Tasks\SonarQube.MSBuild.Tasks.csproj">
```

Comment:
Why aren't you using $(SourcesRoot) here and in line 52 (and why have you stopped using $(SourcesRoot) in the next file?